rollup.config.mjs

```diff
@@ -21,22 +21,17 @@ const modernConfig = {
   ]
 };
 
-const alaskaConfig = {
-  input: {
-    ['auro-alaska__bundled']: './src/auro-alaska.js',
-  },
-  output: {
-    format: 'esm',
-    dir: 'dist/'
-  },
-  plugins: [
-    nodeResolve(),
-    !production &&
-      serve({
-        open: true,
-        openPage: '/docs/'
-      })
-  ]
-};
+function createExampleConfig(entryPoint) {
```

Comment:
I like the idea of the function here, but why do this differently than the other components?

Comparing this to the code from flight:

https://github.com/AlaskaAirlines/auro-flight/pull/137#discussion_r1817088026

Reply:
This was to address the feedback from Sourcery-ai, this is somewhat helpful when we have multiple same structured config. but I can revert it back.